packages/jest-worker/README.md

```diff
@@ -83,12 +83,6 @@ By default, no process is bound to any worker.
 
 The arguments that will be passed to the `setup` method during initialization.
 
-#### `workerPool: (workerPath: string, options?: WorkerPoolOptions) => WorkerPoolInterface` (optional)
```

Comment:
grepped for this and it seems like we don't provide such option, so removed it right away

Reply:
we do? https://github.com/facebook/jest/blob/baafcd48387227d81b1f8b61c9c2881e80e4db67/packages/jest-worker/src/index.ts#L84-L92 Wrong casing, but still

Reply:
Oh my, gonna fix the casing in readme then